packages/replayio/src/utils/mixpanel/trackAsyncEvent.ts

```diff
@@ -0,0 +1,38 @@
+import { debug } from "./debug";
+import { trackEvent } from "./trackEvent";
+import { EventProperties } from "./types";
+
+export async function trackAsyncEvent<Type>(
```

Comment:
We have both `trackAsyncEvent` and `trackEventAsync`. The names make sense and I don't have anything better to propose but it's quite easy to get confused by those names being so similar.

Reply:
Yeah, totally. I had the same thought.